go.mod

```diff
@@ -24,4 +24,6 @@ replace github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alp
 
 replace google.golang.org/grpc => google.golang.org/grpc v1.33.2
 
-replace github.com/tendermint/tendermint => github.com/forbole/tendermint v0.34.13-0.20210820072129-a2a4af55563d
+replace github.com/tendermint/tendermint => github.com/celestiaorg/celestia-core v0.34.14-celestia.0.20211121093520-199a1e4232ac
```

Comment:
while we're at it, we should use the commit after the one listed here, as it has event tracking for malleated txs
266919cced265eb629fd51fdc7ba264f4c3a499a

Reply:
Changed it to: `github.com/celestiaorg/celestia-core v0.34.14-celestia` which should reflect the latest. what do you think ?

Reply:
I don't think that tag includes that change, we could however create a new tag that does. We might also be able to use the branch `v0.34.x-celestia latest`

https://github.com/celestiaorg/celestia-core/commits/v0.34.14-celestia

Reply:
cool then, let's wait until we have all the changes